Syntaxes/Nim.YAML-tmLanguage

```diff
@@ -116,80 +115,67 @@ patterns:
 
 - comment: A float literal
   name: constant.numeric.float.decimal.nim
-  match: (?<![\w\x{80}-\x{10FFFF}])(\d[_\d]*((\.[_\d]+([eE][\+\-]?\d[_\d]*)?)|([eE][\+\-]?\d[_\d]*)))('?([fF](32|64|128))|[fFdD])?   #' highlight-flaw-fixer-comment
```

Comment:
What was wrong with the `[iIuUfF](8|16|32|64)` segments for this line and the others below it?